lib/views/help/requesting.es.html.erb

```diff
@@ -477,22 +474,23 @@
       </p>
       <h2>
-        Los recursos en Nicaragua
+        Los recursos en Nicaragua <a href="#recursos">#</a>
       </h2>
       <p>
-        En caso que la entidad niegue el acceso a la información, entregue información incompleta o no conteste dentro del término establecido por la LAIP, la persona solicitante podrá:
+        En caso que la entidad niegue el acceso a la información, entregue información incompleta o no conteste dentro del término establecido por la LAIP, la persona solicitante podrá (según la Ley de Acceso a la Información Pública y su Reglamento):
       </p>
       <ol>
-        <li>Apelar por escrito o por vía electrónica ante la Coordinación de Acceso a la Información Pública que corresponda, dentro del término de 6 días de notificada la resolución. Esta instancia deberá resolver la apelación en un término de 30 días, agotándose con ella la vía administrativa. Si el escrito de apelación está incompleto, la Coordinación de Acceso a la Información Pública correspondiente deberá completar dichas omisiones.
+        <li>1)	Apelar por escrito o por vía electrónica ante la Coordinación de Acceso a la Información Pública que corresponda, dentro del término de 6 días de notificada la resolución. Esta instancia deberá resolver la apelación en un término de 30 días, agotándose con ella la vía administrativa. Si el escrito de apelación está incompleto, la Coordinación de Acceso a la Información Pública correspondiente deberá completar dichas omisiones. <a href="http://legislacion.asamblea.gob.ni/Normaweb.nsf/($All)/675A94FF2EBFEE9106257331007476F2">(Ver artículos 37 y 38 de la Ley de Acceso a la Información Pública)</a>
+           <p> <a href="http://legislacion.asamblea.gob.ni/Normaweb.nsf/fb812bd5a06244ba062568a30051ce81/7f39c16b8858d606062574020077d8d3?opendocument">Ver artículos 88, 89 y 95 del Reglamento de la Ley de Acceso a la Información Pública.</a> </p>
         </li>
       </ol>
       <p>
-        Ver Arto. 89 del Reglamento de la LAIP.
+          <strong>Nota:</strong>se debe de tener presente que no contamos con información oficial sobre la creación y/o existencia de la Coordinación de Acceso a la Información Pública  en los Poderes del Estado, Gobiernos Regionales Autónomos de la Costa Atlántica y Gobiernos Municipales, cuya función, (según el artículo 13 de la Ley de Acceso a la Información Pública), es velar en el ámbito de su competencia, por el cumplimiento de la presente Ley y constituirse como segunda instancia para conocer y resolver los recursos de apelación que se interpongan contra las resoluciones denegatorias a las solicitudes de acceso a la información pública. Pero eso no te impide seguir adelante demandando una respuesta a tu solicitud; puedes hacer lo siguiente: 
```

Comment:
heh np! You can just put one in, so:

```html
<li>
  <p>Some paragraph stuff</p>
  <p>More paragraph stuff</p>
</li>
```

I'll add the fix in the merge since I've already done it now :)